artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis

```diff
@@ -102,6 +102,9 @@ fi
 
 exec "$JAVACMD" \
     $JAVA_ARGS \
+    --add-opens java.base/java.lang=ALL-UNNAMED \
+    --add-opens java.base/java.io=ALL-UNNAMED \
+    --add-opens java.base/java.util=ALL-UNNAMED \
```

Comment:
Adding the jansi library will remove the need for those additional JVM options.  Those are only required when using the exec provider, which won't be used if you add jansi to support windows.